pypeit/core/telluric.py

```diff
@@ -596,6 +597,46 @@ def general_spec_reader(specfile, ret_flam=False):
 
     return wave, counts, counts_ivar, counts_mask, meta_spec, head
 
+def save_coadd1d_tofits(outfile, wave, flux, ivar, mask, spectrograph=None, telluric=None, obj_model=None,
```

Comment:
@profxj can we simply merge this with coadd1d.CoAdd1D.save by making that method a class method that does not require an instance of the class? You just to let that method take arguments as optional.  These pieces of code are virtually identical. Or maybe this method should be a utility routine in the module where the the CoAdd1d data container is defined, and Coadd1D.save can wrap that.

Reply:
@profxj can you comment on @jhennawi suggestion. I am fine with either keeping the current format or you can help on changing the data container following his suggestion.

Reply:
I'm ok with a static method.

Reply:
Ask @jhennawi to make the change.